apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -2090,6 +2090,10 @@ export const platform: NavMenuConstant = {
           name: 'Your monthly invoice',
           url: '/guides/platform/your-monthly-invoice',
         },
+        {
+          name: 'How to control your costs',
```

Comment:
Maybe shorter without losing any context: Control your costs

Reply:
Yeah, was thinking about that. Was afraid that it might be understood as an "imperative". But you're probably right. Gonna change it.